src/eqProps.js

```diff
@@ -14,7 +14,7 @@ var equals = require('./equals');
  * @param {Object} obj1
  * @param {Object} obj2
  * @return {Boolean}
- *
+ * @deprecated since v0.18.0
```

Comment:
I suggest deprecating [`R.eqProps`][1] since `R.eqProps('x')` is sugar for `R.eqBy(R.prop('x'))`. I'd prefer to promote the use of the more general function in this case.


[1]: http://ramdajs.com/docs/#eqProps